akka-http-marshallers-scala/akka-http-spray-json/build.sbt

```diff
@@ -2,7 +2,6 @@ import akka._
 import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
-AkkaBuild.experimentalSettings
```

Comment:
Hm, perhaps OK to make stable now... Was initially thinking to un-experimental it together with DSL and JSON "streaming". I guess nothing breaking will happen here though, just additions.